### PR TITLE
Fix rsa::load_from_string()

### PR DESCRIPTION
**Pull request by onchere**

### CODE REVIEW

File: src/rsa.cpp [hunk 1]
```diff
@@ -106,6 +106,29 @@ sp_rsa_key rsa::load_from_file(const std::string &path, password_cb on_password)
 	return std::shared_ptr<RSA>(r, ::RSA_free);
 }
 
+struct BIODeleter {
```
Comment (troian) [inline src/rsa.cpp:109]: Mind changing it to `class` and move declaration to `jwtpp.hh` ?

**Review by troian — CHANGES_REQUESTED** [review]
Look awesome. On comment on new type

File: src/rsa.cpp (hunk 1 above)
Comment (onchere) [reply src/rsa.cpp:109]: Will do

**Review by onchere — COMMENTED** [review]
Moved class BIODeleter to jwtpp.hh

**Pull request merged**